phoenicis-scripts/src/main/java/org/phoenicis/scripts/engine/injectors/IncludeInjector.java

```diff
@@ -20,20 +20,22 @@ public IncludeInjector(ScriptFetcher scriptFetcher) {
 
     @Override
     public void injectInto(PhoenicisScriptEngine phoenicisScriptEngine) {
-        final Set<String> includedScripts = new HashSet<>();
+        // store included scripts (include path -> JS object)
+        final Map<String, Object> includedScripts = new HashMap<>();
 
         phoenicisScriptEngine.put("include", (Function<String, Object>) argument -> {
-            final String script = scriptFetcher.getScript(argument);
-            if (script == null) {
-                throwException(new ScriptException(argument + " is not found"));
+            if (!includedScripts.containsKey(argument)) {
+                final String script = scriptFetcher.getScript(argument);
+                if (script == null) {
+                    throwException(new ScriptException(argument + " is not found"));
```

Comment:
`"Script " + argument + " is not found"`

Reply:
Done.